wikimedia/wikipedia-ios
# Native editor - add underline, subscript and superscript formatters

**Phabricator:** Partially https://phabricator.wikimedia.org/T348072

### Notes
* This PR adds the underline, subscript, and superscript formatters to the native editor

### Test Steps
1.  Open the new editor on the staging scheme
2. Confirm you can toggle adding and removing the tags with selected text
3. Confirm the button has selected state when the tag is present
4. Add the tags with no text, confirm you can see the button selected state
5. Confirm you can toggle to remove, and the button is no longer selected
6. Traverse the tags with the cursor and confirm no crashes happen
7. Repeat for each tag
